.nvmrc

```diff
@@ -1 +1 @@
-6.16
+6.16.0
```

Comment:
do we really need to set "0" ?

Reply:
Ok thanks for the comment. I was not aware of this.
Changed approach to compare major and minor versions only.